### PR TITLE
Move from nose to pytest

### PR DESCRIPTION
**Pull request by ernestask**
The original has seen its last commit in 2016. While nose2 exists,
pytest allows writing tests using the assert statement without losing
context.

### CODE REVIEW

**Comment by ernestask** [comment]
Moreover, python-nose might disappear from Fedora: https://lists.fedorahosted.org/archives/list/devel@lists.fedoraproject.org/thread/NNGYNRUH5VBA2MO57C3VE3ON4QFAPEKR/

File: src/python-problem/tests/Makefile.am [hunk 1]
```diff
@@ -4,11 +4,8 @@ check_SCRIPTS = $(TESTS)
...
-check-local:
-	echo "Running tests with python3"; \
-	exc=0; \
-	for test in $(TESTS); do \
-		python3 $$test || exc=1; \
-	done; \
-	exit $$exc
+if HAVE_PYTEST
+check-local: $(TESTS)
+	pytest-3 --cov=problem --capture=no --verbose $(TESTS)
```
Comment (mkutlak) [inline src/python-problem/tests/Makefile.am]:
```
ERROR: usage: pytest-3 [options] [file_or_dir] [file_or_dir] [...]                                                     
pytest-3: error: unrecognized arguments: --cov=problem                                                                                                                                                                                        
  inifile: None                                                                                                        
  rootdir: /home/mkutlak/ABRT/abrt/build/abrt-2.12.2.67.gb050/src/python-problem/tests
```

Comment (ernestask) [reply src/python-problem/tests/Makefile.am]: Ah, good catch, I think it’s a plugin. Better to not care about coverage here, then, but rather in Jenkins.

**Comment by mkutlak** [comment]
image

LGTM

**Pull request merged**